Doc/library/os.rst

```diff
@@ -853,6 +872,8 @@ as internal buffering of data.
    .. versionchanged:: 3.4
       The new file descriptor is now non-inheritable.
 
+   .. availability:: not WASI.
```

Comment:
```suggestion
   .. availability:: not WASI.
   
   .. versionchanged:: 3.4
      The new file descriptor is now non-inheritable.
```

In almost all the existing usages in this module (and essentially all those I checked elsewhere) as well as most of those added here, ``availability`` is always listed above ``versionchanged``. Let's be consistent here, especially since this is Reference-focused documentation, as a clear and consistent structure helps readers find the key information they are looking for easily without having to dig around for it.

Reply:
I have moved the availability information above version changed.